game/scripts/npc/heroes/techies.txt

```diff
@@ -5,9 +5,9 @@
   //=================================================================================================================
   "npc_dota_hero_techies"
   {
-    "Ability13"                                           "special_bonus_intelligence_35" // replaces special_bonus_mp_regen_3
-
     "AttackDamageMin"                                     "24" // 9
     "AttackDamageMax"                                     "26" // 11
-  }
+    "Ability13"                                           "special_bonus_all_stats_15" // replaces special_bonus_mp_regen_3
```

Comment:
Opted to keep the theme however and gave him stats. 15 strength will make techies a little more tanky in that mid-game and the agility + intelligence will allow techies's physical build to be a little more viable as well (more viable than the 35 int).

Reply:
Hero with highest attack range doesn't need to be tanky.

Reply:
> Hero with highest attack range doesn't need to be tanky.

A hero who's major source of dmg (only spell with amp) being diving in to the enemy team should have some semi-tanky talent. If you want me to make this 10 stats I am ok with that but 15 strength isn't going to break the camel back. (You gave meepo 30 strength at level 10 remember XD)

Also most people will choose other talent anyway.